src/main/java/com/faforever/api/player/PlayerService.java

```diff
@@ -17,7 +18,9 @@
 
   private final PlayerRepository playerRepository;
 
-  public Player getPlayer(Authentication authentication) {
+  public Player getCurrentPlayer() {
+    Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
I am not sure if UserSupplier could be used here, or it is being redundant due to this change

Reply:
I wouldn't know a case where you have User instead of a Player except for the current logged in one.